geode-core/src/main/java/org/apache/geode/distributed/internal/PooledExecutorWithDMStats.java

```diff
@@ -158,8 +159,8 @@ public PooledExecutorWithDMStats(BlockingQueue<Runnable> q, int poolSize, PoolSt
    * Default timeout with no stats.
    */
   public PooledExecutorWithDMStats(BlockingQueue<Runnable> q, int poolSize, ThreadFactory tf,
-      ThreadsMonitoring tMonitoring) {
-    this(q, poolSize, null/* no stats */, tf, tMonitoring);
+      ThreadsMonitoring tMonitoring, String systemPropertyPrefix) {
```

Comment:
I'm not sure I see the reason for this change - doesn't look like this class moved out of core?

Reply:
It stems from another effort to break deps on CoreLoggingExecutors, I'd like to leave the change since it now uses `SystemPropertyHelper.GEMFIRE_PREFIX` which seems the more appropriate constant to be used...